scripts/docker_build/sof_qemu/Dockerfile

```diff
@@ -22,8 +22,13 @@ ARG host_https_proxy
 ENV http_proxy $host_http_proxy
 ENV https_proxy $host_https_proxy
```

Comment:
remove for QEMU image too?

Reply:
Share some of the code to avoid copy/paste/diverge?

Reply:
Removed for QEMU images. But for sharing same code between two images, I don't have much idea right now. I'd like to do with new issue.

Reply:
> But for sharing same code between two images, I don't have much idea right now

OK, some other time then.